dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/processor/TaskKillResponseProcessor.java

```diff
@@ -51,7 +51,8 @@ public void process(Channel channel, Command command) {
         Preconditions.checkArgument(CommandType.TASK_KILL_RESPONSE == command.getType(), String.format("invalid command type : %s", command.getType()));
 
         TaskKillResponseCommand responseCommand = JSONUtils.parseObject(command.getBody(), TaskKillResponseCommand.class);
-        logger.info("received task kill response command : {}", responseCommand);
+        logger.info("[TaskInstance-{}] Received task kill response command : {}",
```

Comment:
Looks like this class/method does nothing but just log the message?

Reply:
Yes, this might be a bug, we may need to judge the kill status and update the database. 
One situation is that we believe it kill success, but it still running.
cc @caishunfeng